Branch-SDK/src/io/branch/referral/InstantAppUtil.java

```diff
@@ -68,14 +68,10 @@ private static boolean isPreReleaseOBuild() {
     @SuppressWarnings("ConstantConditions")
     static boolean doShowInstallPrompt(@NonNull Activity activity, int requestCode, @Nullable String referrer) {
         if (activity == null) {
-            if (Branch.isLogging_ != null && Branch.isLogging_) {
-                Log.e("BranchSDK", "Unable to show install prompt. Activity is null");
-            }
+            Log.e("BranchSDK", "Unable to show install prompt. Activity is null");
```

Comment:
Why are we logging without checking that debug mode is enabled?

Reply:
If it is truly an error, it should always be logged.   These two instances of logging were the only ones in the entire code base that went directly to `Log` instead of going through the Branch Log method.

Reply:
Outdated.